Abracadabra/abracadabra_admin_panel/pages/expertverification.js

```diff
@@ -1,30 +1,89 @@
 //react
 import React from 'react';
-import {useEffect} from 'react';
+import {useEffect, useState} from 'react';
 import { signIn, signOut, useSession } from 'next-auth/client';
 //bootstrap
 import { Container, Row, Col } from 'react-bootstrap';
 //components
 import NavBar from 'src/components/NavBar';
-import SideBar from 'src/components/SideBar'
-import {} from 'react-icons/fa'
+import SideBar from 'src/components/SideBar';
+import {} from 'react-icons/fa';
+import {useDate} from 'src/components/Dashboard/Dashboard_Greeting';
+import Dashboard_ExpertCard from 'src/components/Dashboard/Dashboard_ExpertCard';
+import ExpertService from 'src/services/ExpertService';
 
 export default function Expertverification() {
   const [ session, loading ] = useSession();
+  // const router = useRouter();
+  const { date, time, wish } = useDate();
+  const [ExpertApplications, setExpertApplications] = useState([]);
+
+  function RemoveApplication(id){
+    console.log(id);
+    let apps = ExpertApplications;
+    const arr = apps.filter((item) => item.applicationId !== id);
+    setExpertApplications(arr);
+  };
+
+  useEffect(() => {
+    //Api call applications
+    ExpertService.GetApplications()
+    .then((res) => {
+      console.log(res);
+      setExpertApplications(res.data);
+    })
+    .catch((error) => {
+       console.log(error);
```

Comment:
May want to display a message the admin can see, without opening the console. This is for the release.

Reply:
Yes definitly